### DISCUSSION TITLE
2023/books/cormac/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/shane0.github.io#56] [General]
# 2023/books/cormac/

https://shanenull.com/2023/books/cormac/

**Comment by shane0** [comment]
outer dark
https://youtu.be/0WSzN3k0axs?si=pHcXz7bDykY2gK8m

**Comment by shane0** [comment]
“Ultimately there is nothing to know and no one to know it.”
― Cormac McCarthy, The Passenger

**Comment by shane0** [comment]
The problem is that what drives the tale will not survive the tale. As the room dims and the sound of voices fades you understand that the world and all in it will soon cease to be.

**Comment by shane0** [comment]
nlp sample rewrite

The sun beat down on the dusty town of Agua Caliente, turning the adobe bricks into ovens and shimmering the air like a mirage. A tumbleweed, the only thing moving besides the sun, snagged on the hitching post in front of the cantina. Inside, the air was thick with the smell of sweat, stale beer, and something metallic – blood maybe.

Two men sat hunched over their drinks, backs to each other. One, a weathered gunslinger named Jebidiah, wore a dusty duster and a wide-brimmed hat that cast his face in shadow. The other, a nameless bandit with a greasy scar down his cheek, nursed a mescal, his eyes flicking nervously around the room.

The silence stretched, broken only by the buzz of flies. Then, a creak of the swinging doors. A tall figure, all black against the harsh sunlight, filled the doorway. It was the Judge, a man whispered about in hushed tones, a killer with a reputation as cold as the desert wind. His eyes, like chips of obsidian, found Jebidiah and the bandit. A ghost of a smile played on his lips.

"Well, well," the Judge drawled, his voice a gravelly rasp. "Look what the sun drug in. Heard there was trouble brewing in Agua Caliente."

Jebidiah didn't move, but his hand twitched near the hilt of the long knife strapped to his thigh. The bandit, however, choked on his mescal, his fear a palpable thing in the air.

The Judge sauntered towards their table, his boots crunching on the dusty floorboards. He stopped in front of the bandit, his gaze like a rattler sizing up its prey.

"You got something of mine," the Judge said, his voice low and dangerous.

The bandit whimpered, shaking his head. "N-nothing, sir. Not a thing."

The Judge didn't reply. He just reached out, lightning fast, and snatched the bandit's knife from its scabbard. The blade gleamed wickedly in the dim light.

"This yours?" The Judge flicked the knife, catching the light on the engraved inscription on the hilt – a silver rattlesnake.

Jebidiah slammed his fist on the table, the sound echoing in the tense silence. "That's mine, you sidewinder!"

The Judge's smile widened, a cruel slash of white against his dark beard. "Seems we got ourselves a little disagreement here, boys."

He tossed the knife in the air, catching it with a flourish. In the same motion, he slashed across the bandit's throat. A crimson arc erupted, staining the air. The bandit gurgled, clutching at his wound, before crumpling to the floor, dead before he hit the ground.

Jebidiah was already on his feet, his own knife singing through the air. The Judge met him head-on, their blades clashing in a shower of sparks. The fight was a whirlwind of steel and fury. Jebidiah, fueled by rage and years of honing his skills, fought like a demon. The Judge, however, was a predator, calm and calculating, his movements deadly precise.

They danced a bloody ballet across the cantina floor, furniture splintering as they collided. Jebidiah landed a slash across the Judge's arm, drawing a growl of pain. The Judge retaliated, his blade finding a gap in Jebidiah's defenses, carving a deep gash across his chest.

Jebidiah stumbled back, gasping for breath. Blood soaked his shirt, staining the dusty floor crimson. He knew he was losing, his vision blurring at the edges. But he wouldn't give up. Not without a fight.

With a final surge of adrenaline, Jebidiah lunged. The Judge sidestepped, his knife a blur of silver. It sank deep into Jebidiah's gut, a wet, sickening sound. Jebidiah stared at the blade in disbelief, then down at his own blood staining the Judge's black coat.

The Judge slowly withdrew his knife, wiping it on Jebidiah's dying shirt. "Looks like you lose, old man," he said, his voice devoid of emotion.

Jebidiah crumpled, his life draining away with each ragged breath. The cantina was silent, the only sound the rasping coughs of a dying man. The Judge surveyed the scene, a dark silhouette against the dying light filtering through the windows. 

Then, with a flick of his wrist, he sent Jebidiah's silver rattlesnake knife spinning through the air. It landed on the table with a clatter, a grim trophy in the wake of carnage.  The sun dipped below the horizon, painting the sky in fiery hues of red and orange, a fitting end to a bloody day in Agua Caliente.

**Comment by shane0** [comment]
https://github.com/shane0/stoicism/issues/13

**Comment by shane0** [comment]
https://shanenull.com/stoicism/cormac/

**Comment by shane0** [comment]
# chatbot errors

- as mentioned this version of this site contains chatbot responses
- this quote was not cormac

```text
“The world keeps ending but new people too dumb to know it keep showing up as if the fun's just started.”
-  John Updike, Rabbit Is Rich
```

- https://www.goodreads.com/quotes/143616-the-world-keeps-ending-but-new-people-too-dumb-to
